src/libs/actions/PolicyConnections.ts

```diff
@@ -0,0 +1,87 @@
+import Onyx from 'react-native-onyx';
+import type {OnyxUpdate} from 'react-native-onyx';
+import * as API from '@libs/API';
+import type UpdateConnectionConfigParams from '@libs/API/parameters/UpdatePolicyConnectionConfigurationParams';
+import {WRITE_COMMANDS} from '@libs/API/types';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {ConnectionName, Connections} from '@src/types/onyx/Policy';
+
+function updatePolicyConnectionConfig<TConnectionName extends ConnectionName, TSettingName extends keyof Connections[TConnectionName]['config']>({
```

Comment:
Shouldn't you be deleting the version of this function that we have here: https://github.com/Expensify/App/blob/7398f755820e67bea89b95886e5eab3007c162d8/src/libs/actions/Policy.ts#L3724-L3746

or do you have a plan to do that in a separate PR?

While working on https://github.com/Expensify/App/pull/39219 I found some stuff that was duplicated already, like the QBO "Set up" button component. I think we should take better care of not having double implementations of stuff.

Reply:
yes I saw that Nikolay has implemented this action function already

I instructed CallStack engineers that these are handled separately but they implemented anyway 😓